packages/vuetify/package.json

```diff
@@ -33,7 +33,6 @@
   "module": "dist/vuetify.js",
   "jsdelivr": "dist/vuetify.js",
   "unpkg": "dist/vuetify.js",
-  "types": "types/index.d.ts",
```

Comment:
?

Reply:
Types blew everything up, like really really bad, caused the dev server to crash after reporting about 1 trillion lint errors.

Reply:
dev seems to work fine with it, the errors on prod build are because of everything else that's changed

Reply:
I'll try again but even in dev this was happening.

Reply:
Ok just adding back types didn't change anything, the errors in dev are from eslint-loader and ForkTsCheckerWebpackPlugin